tools/mggo/tools.go

```diff
@@ -9,16 +9,13 @@ import (
 	"go.einride.tech/mage-tools/mgtool"
 )
 
-func Command(args ...string) *exec.Cmd {
-	return mgtool.Command("go", args...)
-}
-
 func TestCommand() *exec.Cmd {
 	coverFile := mgpath.FromTools("go", "coverage", "go-test.txt")
 	if err := os.MkdirAll(filepath.Dir(coverFile), 0o755); err != nil {
 		panic(err)
 	}
-	return Command(
+	return mgtool.Command(
+		"go",
```

Comment:
What is your thoughts on adding the `-shuffle on` flag?

Reply:
Great idea!! Adding

Reply:
Just note that it may break tests on some projects which may cause some friction :sweat_smile: